services/src/northbound/src/main/java/org/openkilda/northbound/converter/SwitchMapper.java

```diff
@@ -14,18 +30,10 @@
 
     SwitchDto toSwitchDto(SwitchInfoData data);
 
-    default RulesSyncResult toRulesSyncResult(SyncRulesResponse response) {
-        return new RulesSyncResult(response.getMissingRules(), response.getProperRules(),
-                response.getExcessRules(), response.getInstalledRules());
-    }
-
     default RulesSyncResult toRulesSyncResult(RulesValidationResult validationResult, List<Long> installedRules) {
         return new RulesSyncResult(validationResult.getMissingRules(), validationResult.getProperRules(),
                 validationResult.getExcessRules(), installedRules);
     }
 
-    default RulesValidationResult toRulesValidationResult(SyncRulesResponse response) {
-        return new RulesValidationResult(response.getMissingRules(), response.getProperRules(),
-                response.getExcessRules());
-    }
+    RulesValidationResult toRulesValidationResult(SyncRulesResponse response);
```

Comment:
Doesn't it make sense to move the method into a dedicated mapper class? E.g. RulesValidationMapper

Reply:
Maybe. I don't exactly know how it is logically corresponds with switches